pgxn/neon/neon.c

```diff
@@ -30,9 +31,68 @@
 PG_MODULE_MAGIC;
 void		_PG_init(void);
 
+static ProcessUtility_hook_type PreviousProcessUtilityHook = NULL;
+
+/*
+ * Utulity hook handling creation of Lsm3 indexes
```

Comment:
```suggestion
 * Utility hook handling creation of Lsm3 indexes
```

Reply:
afaict the entire comment is incorrect/misleading?